<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MenuDialog</class>
 <widget class="QDialog" name="MenuDialog">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>844</width>
    <height>527</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Список записей</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="lblPeriod">
       <property name="text">
        <string>Период с</string>
       </property>
       <property name="buddy">
        <cstring>edtBegDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtBegDate">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFor">
       <property name="text">
        <string>по</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
       <property name="buddy">
        <cstring>edtEndDate</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CDateEdit" name="edtEndDate">
       <property name="displayFormat">
        <string>dd.MM.yyyy</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="lblFeaturesToEat">
       <property name="text">
        <string>Особенности</string>
       </property>
       <property name="buddy">
        <cstring>edtFeaturesToEat</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="edtFeaturesToEat"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Финансирование</string>
       </property>
       <property name="buddy">
        <cstring>cmbFinance</cstring>
       </property>
      </widget>
     </item>
     <item>
      <widget class="CRBComboBox" name="cmbFinance">
       <property name="minimumSize">
        <size>
         <width>140</width>
         <height>0</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkRefusalToEat">
       <property name="text">
        <string>Отказ</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="chkUpdate">
       <property name="text">
        <string>Обновить</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="CTableView" name="tblItems">
     <property name="whatsThis">
      <string comment="ура!">список записей</string>
     </property>
     <property name="tabKeyNavigation">
      <bool>false</bool>
     </property>
     <property name="alternatingRowColors">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout">
     <property name="spacing">
      <number>4</number>
     </property>
     <property name="margin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>всего: </string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStatusBar" name="statusBar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="toolTip">
        <string>A status bar</string>
       </property>
       <property name="whatsThis">
        <string>A status bar.</string>
       </property>
       <property name="sizeGripEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer>
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnSelected">
       <property name="text">
        <string>Выбрать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnEdit">
       <property name="whatsThis">
        <string>изменить текущую запись</string>
       </property>
       <property name="text">
        <string>Просмотр</string>
       </property>
       <property name="shortcut">
        <string>F4</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnPrint">
       <property name="text">
        <string>Печать</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="whatsThis">
        <string>выйти из списка без выбора</string>
       </property>
       <property name="text">
        <string>Закрыть</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CDateEdit</class>
   <extends>QDateEdit</extends>
   <header>library/DateEdit</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CTableView</class>
   <extends>QTableView</extends>
   <header>library/TableView.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>edtBegDate</tabstop>
  <tabstop>edtEndDate</tabstop>
  <tabstop>edtFeaturesToEat</tabstop>
  <tabstop>cmbFinance</tabstop>
  <tabstop>chkRefusalToEat</tabstop>
  <tabstop>chkUpdate</tabstop>
  <tabstop>tblItems</tabstop>
  <tabstop>btnSelected</tabstop>
  <tabstop>btnEdit</tabstop>
  <tabstop>btnPrint</tabstop>
  <tabstop>btnCancel</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnCancel</sender>
   <signal>clicked()</signal>
   <receiver>MenuDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>836</x>
     <y>519</y>
    </hint>
    <hint type="destinationlabel">
     <x>119</x>
     <y>449</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
